shared/charts/couchdb-prometheus-exporter/README.md

```diff
@@ -48,10 +48,12 @@ Parameter | Description | Default
 --- | --- | ---
 `replicaCount` | desired number of pods | `1`
 `exporterListenPort` | port for exporter service to listen on | `9984`
-`couchdbUri` | URI for couchdb | `http://couchdb-svc-couchdb.default.svc.cluster.local:5984`
-`couchdbUsername` | username for couchdb uri | `admin`
-`couchdbPassword` | password for couchdb uri | `password`
-`couchdbDatabases` | list of specific databases to monitor | `_all_dbs`
+`couchdb.labels` | CouchDB pod labels for `NetworkPolicy` | `{ app: "couchdb" }`
+`couchdb.port` | CouchDB port for `NetworkPolicy` | `http-couchdb`
```

Comment:
I understand what this is, but when I see `couchdb.port`, I expect to find a value like `5984`. How do you feel about `portName` or `podPortName` or something? Maybe `port` being a name isn't confusing to others in this Kubernetes context where ports are often names?

Reply:
I kept the name `port` as both forms are accepted here, but improved comment by adding `(can either be a numerical or named port on a pod)` to this and all the other charts.